doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml

```diff
@@ -663,8 +753,7 @@ If `Multi Subnet Failover` is set to `true`, this setting has no effect.
   
 |Key|Default value|  
 |---------|-------------------|  
-|Application Name|".Net SqlClient Data Provider"|  
-|Asynchronous Processing(Obsolete)|False|  
+|Application Name|"Core Microsoft SqlClient Data Provider"|  
```

Comment:
```suggestion
|Application Name|"Framework Microsoft SqlClient Data Provider" when running on .NET Framework. "Core Microsoft SqlClient Data Provider" otherwise.|  
```